lib/views/pages/profile/user_info/user_account.dart

```diff
@@ -3,14 +3,13 @@ import "package:flutter/material.dart";
 /// This widget display the user info in the profile page
 class UserAccount extends StatelessWidget {
   static const userInfoKey = Key("user Info");
+  static const centauriPointsKey = Key("centauri points");
```

Comment:
Just applying camelCase :+1: 
```suggestion
  static const userInfoKey = Key("userInfo");
  static const centauriPointsKey = Key("centauriPoints");
```